webapp/packages/plugin-authentication-administration/src/Administration/Users/Teams/GrantedUsers/GrantedUsers.tsx

```diff
@@ -55,22 +55,18 @@ export const GrantedUsers: TabContainerPanelComponent<ITeamFormProps> = observer
 
   if (isDefaultTeam) {
     return (
-      <Container className={s(styles, { box: true })} parent gap vertical>
-        <Group className={s(styles, { placeholderBox: true })} keepSize large>
-          <TextPlaceholder>{translate('plugin_authentication_administration_team_default_users_tooltip')}</TextPlaceholder>
```

Comment:
its probably better to keep TextPlaceholder component as if we would like to make changes to all placeholders in the app, we cant do it if we use Group